Spin-Up/HSKY15_SpinUp/src/motion/motionprofilepoint.cpp

```diff
@@ -0,0 +1,21 @@
+#include "motionprofilepoint.hpp"
+
+ControlVector MotionProfilePoint::getControlVector() {
+    return this->controlVector;
+}
+
+float MotionProfilePoint::getCurve() {
+    return this->curve;
+}
+
+float MotionProfilePoint::getLeftVel() {
+    return this->leftVel;
+}
```

Comment:
Let's try to avoid over-simplifying variable names. This can be named verbosely `leftChassisVelocity`